test/e2e/framework/volume/fixtures.go

```diff
@@ -155,8 +156,11 @@ func NewNFSServer(cs clientset.Interface, namespace string, args []string) (conf
 	if len(args) > 0 {
 		config.ServerArgs = args
 	}
-	pod, ip = CreateStorageServer(cs, config)
-	return config, pod, ip
+	pod, host = CreateStorageServer(cs, config)
+	if strings.Contains(host, ":") {
```

Comment:
Is there any chance hosts will be passed in with a port suffixed?  If so, will wrapping an ipv4 address with `[]` be a bug?